manifests/facts.pp

```diff
@@ -46,7 +50,8 @@
     }
   }
 
-  if $::osfamily == 'RedHat' and $::puppetversion !~ /Puppet Enterprise/ {
+  # Install $json_packages only if osfamily is RedHat and agent is version PE 3.8 and less or not an AIO version 1.3.x
+  if $::osfamily == 'RedHat' and (($::puppetversion !~ /Puppet Enterprise/) and ($::aio_agent_version !~ /1\.3\.*[0-9]*/)) {
```

Comment:
Here I wasn't sure where the "Puppet Enterprise" comparaison was coming from, so I decided in the name of compatibility to keep that test.